core/src/main/resources/lib/test/bar_fr.properties

```diff
@@ -22,5 +22,5 @@
 
 No\ tests=Pas de test
 failures={0} \u00E9checs
-skipped=Non pass�s
+skipped={0] non pass�s
```

Comment:
Needs to be a close curly brace.

Reply:
Hi Daniel, yes I made a mistake when I first push so I have pushed force a new commit with the ending curly  brace.
What should I do?